HW2/P3/P3.txt

```diff
@@ -0,0 +1,32 @@
+Without Instruction-level Parallelism
+
+1 thread:
+  1074.656613 Million Complex FMAs in 6.87227201462 seconds,
+  156.375738724 million Complex FMAs / second
+
+2 threads:
+  1074.656613 Million Complex FMAs in 3.39673495293 seconds,
+  316.379295969 million Complex FMAs / second
+
+4 threads:
+  1074.656613 Million Complex FMAs in 1.82714009285 seconds,
+  588.163226895 million Complex FMAs / second
+
+
+With Instruction-level Parallelism
+
+1 thread:
```

Comment:
This looks too good to be true. Mine is 1074.926712 Million Complex FMAs in 0.620010852814 seconds. The performance improves, but looks more realistic.